docs/user-guide/getting-started.md

```diff
@@ -11,175 +11,189 @@ The Ambassador Edge Stack is designed to run in Kubernetes for production. The m
+ -> Remote Kubernetes cluster detected.
+ -> Provisioning a cloud load balancer. (This may take a minute, depending on your cloud provider.)
+ -> Automatically configuring TLS.
+ Please enter an email address. We’ll use this email address to notify you prior to domain and certification expiration [None]: john@example.com.
+```
 
-    ```bash
-    (⎈ |minikube:ambassador)$ minikube service list
-    |-------------|------------------|--------------------------------|
-    |  NAMESPACE  |       NAME       |              URL               |
-    |-------------|------------------|--------------------------------|
-    | ambassador  | ambassador       | http://192.168.64.2:31230      |
-    |             |                  | http://192.168.64.2:31042      |
-    | ambassador  | ambassador-admin | No node port                   |
-    | ambassador  | ambassador-redis | No node port                   |
-    | default     | kubernetes       | No node port                   |
-    | kube-system | kube-dns         | No node port                   |
-    |-------------|------------------|--------------------------------|
-    ```
+6. Provide an email address as required by the ACME TLS certificate provider, Let's Encrypt. Then your terminal will print something similar to the following:
 
-    Use any of the URLs listed next to `ambassador` to access the Ambassador Edge Stack.
+```shell
+ -> Obtaining a TLS certificate from Let’s Encrypt.
 
-3. Navigate to `http://<your-IP-address>` and click through the certificate warning for access to the Edge Policy Console interface. The certificate warning appears because, by default, the Ambassador Edge Stack uses a self-signed certificate for HTTPS.
-    * Chrome users should click **Advanced > Proceed to website**.
-    * Safari users should click **Show details > visit the website** and provide your password.
+ Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit https://random-word-3421.edgestack.me to access your Edge Stack installation and for additional configuration.
+ ```
 
-4. To login to the [Edge Policy Console](../../about/edge-policy-console), download and install `edgectl`, the command line tool Edge Control, by following the provided instructions on the page. The Console lists the correct command to run and provides download links for the `edgectl` binary.
+Your new [Edge Policy Console](/about/edge-policy-console) will open
+automatically in your browser at the provided URL. **Note that the provided `random-word.edgestack.me` domain name will expire after 90 days**.
 
-  The Edge Policy Console must authenticate your session using a Kubernetes Secret in your cluster. Edge Control accesses that secret using `kubectl`, then sends a URL to your browser that contains the corresponding session key. This extra step ensures that access to the Edge Policy Console is just as secure as access to your Kubernetes cluster.
+### Minikube Users
```

Comment:
so does this still belong here?

Reply:
In our original quick start (before AES) we had minikube user instructions inline. So i'd argue yes.

Reply:
@scoyle391 Not sure I follow. Why do we still need minikube user instructions with the edgectl install changes?